index.js

```diff
@@ -88,13 +88,13 @@ module.exports = function (opts) {
 
   async function parseBody(ctx) {
     if (enableJson && ((detectJSON && detectJSON(ctx)) || ctx.request.is(jsonTypes))) {
-      return await parse.json(ctx, jsonOpts);
+      return await parse.json(ctx, Object.assign({}, jsonOpts));
```

Comment:
we don't need to clone options in every request. co-body may change the options, but it always change to the same value.